CONTRIBUTING.md

```diff
@@ -105,7 +105,12 @@ We don't follow any other convention, but if you want to use one, we recommend [
 
 ### Changesets <!-- omit in TOC -->
 
-We use [changesets](https://github.com/Noviny/changesets) to do versioning. What that means is that you need to add a changeset by running `yarn changeset` which contains what packages should be bumped, their associated semver bump types and some markdown which will be inserted into changelogs.
+We use [changesets](https://github.com/Noviny/changesets) to do versioning. What that means is that you need to [add a changeset](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) by running `yarn changeset` which contains what packages should be bumped, their associated semver bump types and some markdown which will be inserted into changelogs.
```

Comment:
`and some markdown which will be inserted into changelogs`
Maybe we can formulate it differently, like `and the changelogs of your choice in the Markdown format`? I don't really like the `some markdown` sentence because we don't really know what we're talking about 🤷‍♀️

Reply:
I simplified it a lot

Reply:
We use [changesets](https://github.com/Noviny/changesets) to do versioning. What that means is that you need to [add a changeset](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) by running `yarn changeset`. The changeset you create contains what package should be bumped, its associated semver type and the changelogs.